doc/dev: Postgres upgrade retrospective (#2336)
sourcegraph/sourcegraph: doc/dev/retrospectives/postgres_upgrade.md

# Tomás’ notes on the Postgres upgrade

## Action Items / Learnings

1. Master must always be in a releasable state. Don’t create breaking changes that block the next release.
1. Estimate and plan work before scheduling it for a release.

> 1. Default to use Github Issues to share context on work done (over Slack).

Reviewer: ```suggestion
1. Prefer to share context using Github Issues instead of Slack.
```

> 1. Warning signs to be careful with adopted issues: Missed estimates, being passed around, and fixed deadline.

Reviewer: ```suggestion
1. Warning signs to be careful with adopted issues: missed estimates, being passed around, and fixed deadline.
```
